stable/vulcan/values.yaml

```diff
@@ -151,6 +156,7 @@ persistence:
     snsTopic: arn:aws:sns:local:012345678900:VulcanK8SChecks
     nessusCheckQueue: VulcanK8SChecksTenable
     s3Scans: scans
+    awsCreateChecksSqsUrl: https://sqs.eu-west-1.amazonaws.com/012345678900/VulcanK8SPersistenceChecks
```

Comment:
I think this URL won't work with the internal infra. 
I suggest make the persistence component configurable with arn instead of URL endpoints like the other components.